railties/lib/rails/commands/console/console_command.rb

```diff
@@ -52,10 +52,10 @@ def start
       set_environment! if environment?
 
       if sandbox?
-        puts "Loading #{Rails.env} environment in sandbox (Rails #{Rails.version})"
-        puts "Any modifications you make will be rolled back on exit"
+        say "Loading #{Rails.env} environment in sandbox (Rails #{Rails.version})"
```

Comment:
We don't have `say` here.

Reply:
I see https://travis-ci.org/rails/rails/jobs/419310370

Reply:
Fixed